test/types/defineComponent.spec.ts

```diff
@@ -144,6 +144,17 @@ describe('defineComponent', () => {
     expect.assertions(2);
   });
 
+  it('should accept tuple props', () => {
+    const App = createComponent({
```

Comment:
Do you mean `defineComponemt`?

Reply:
Yes, just fixed. I also updated the type of createComponent